.github/workflows/nightly.yml

```diff
@@ -190,3 +193,26 @@ jobs:
           asana-section: ${{ vars.GH_ANDROID_APP_RELEASES_PROJECT_BLOCKER_SECTION_ID }}
           asana-task-id: ${{ steps.create-failure-task.outputs.taskId }}
           action: 'add-task-asana-project'
+
+  tag_lgc_when_successful:
+    name: Tag last good commit when successful
+    runs-on: ubuntu-24.04 #https://github.com/actions/runner-images/issues/6709
+    needs: [ code_formatting, unit_tests, lint, android_tests ]
+    if: ${{ success() }}
+    steps:
+      - name: Set up git config
+        run: |
+          git remote set-url origin https://${{ secrets.GT_DAXMOBILE }}@github.com/duckduckgo/Android.git/
+          git config --local user.email "github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Create tag
+        run: |
+          TAG_NAME="nightly-$(date -u +'%Y-%m-%dT%H%M%S')"
```

Comment:
I don’t like the idea of nightly as a tag for releases, why not LGC as you have in the proposal?